File: Códigos Aula Cecília/Car.java
// Classe car
//package classe;

public class Car {
    // Atributos da Classe Car
    public String model;
    public String brand;
    public String year;
    public String price;

    // Métodos da Classe Pessoa
    public void km0(){
        System.out.println("Rodagem: 0 km!\n");
    }

    public void semiNew(){
        System.out.print("Rodagem: 1 - 999Km!\n");
    }

    public void used(){
    System.out.print("Rodagem: >1000Km!\n");
    }

    public void showCar(){
        System.out.println("Carro: \n- " + model + "\n- " + brand + "\n- " + year + "\n- " + price);
    }
}

File: Códigos Aula Cecília/car.py
class Car:


    def __init__(self, model, brand, year, price):
        self.price = price
        self.year = year
        self.brand = brand
        self.model = model


    def Km0(self):
        print('Rodagem: 0 km!')

    def SemiNew(self):
        print('Rodagem: 1 - 999Km!')

    def Used(self):
        print('Rodagem: >1000Km!')

    def ShowCar(self):
        print('Carro: \n- {}\n- {}\n- {}\n- {}'.format(self.model, self.brand, self.year, self.price))
